src/commands/projects/accounts/edit.ts

```diff
@@ -11,41 +11,47 @@ export default class Edit extends Command<{success: boolean; warnings?: string[]
 
   static flags = {
     'add-owner': Flags.string({
-      description: 'ID of user to add as an owner.',
+      description: 'ID or email of user to add as an owner.',
       multiple: true,
     }),
     'remove-owner': Flags.string({
-      description: 'ID of user to remove as an owner',
+      description: 'ID or email of user to remove as an owner',
       multiple: true,
     }),
     'add-editor': Flags.string({
-      description: 'ID of user to add as an editor.',
+      description: 'ID or email of user to add as an editor.',
       multiple: true,
     }),
     'remove-editor': Flags.string({
-      description: 'ID of user to remove as an editor',
+      description: 'ID or email of user to remove as an editor',
       multiple: true,
     }),
     'add-analyst': Flags.string({
-      description: 'ID of user to add as an analyst.',
+      description: 'ID or email of user to add as an analyst.',
       multiple: true,
     }),
     'remove-analyst': Flags.string({
-      description: 'ID of user to remove as an analyst',
+      description: 'ID or email of user to remove as an analyst',
       multiple: true,
     }),
     'add-viewer': Flags.string({
-      description: 'ID of user to add as a viewer.',
+      description: 'ID or email of user to add as a viewer.',
       multiple: true,
     }),
     'remove-viewer': Flags.string({
-      description: 'ID of user to remove as an viewer',
+      description: 'ID or email of user to remove as an viewer',
       multiple: true,
     }),
     'project-id': Flags.string({
-      description: 'ID of project to read. Overrides apimetrics config project set.',
+      description:
+        'ID of project to read. Overrides apimetrics config project set. This must be in the specified organization.',
       char: 'p',
     }),
+    'org-id': Flags.string({
+      description:
+        'ID of organization to read. Overrides apimetrics config org set. This must match the organization the project is part of.',
+      char: 'o',
+    }),
```

Comment:
@ndenny I am just wondering, it would theoretically be possible for a user to set the org ID to be different to the org the specific project is in. We could make an extra call to get the project and use the org that that is in? What do you think?